src/en/community/admin/admin-tooling.md

```diff
@@ -1,135 +1,175 @@
 
 The *Round* tab lets you control the flow of the current round. You can start, end, or restart the round using this menu. There are equivalent commands for all of these.
 
 The *Server* tab lets you shutdown/restart the entire server, or toggle OOC/LOOC. This is only available to admins with the proper flags.
 
-The *Player* tab lets you view all of the players in a round. You can also toggle the **Player Overlay** here, which draws some basic information (ckey, character name, is antag) over all players in game.
+The *Panic Bunker* tab lets you toggle settings for the Panic Bunker, such as enabling or disabling the Panic Bunker, automatically enabling or disabling the Panic Bunker, count deadmined players, and showing the reason why the Panic Bunker is enabled. You can also set the minimum required account age or overall playtime to bypass the Panic Bunker.
+
+The *Player* tab lets you view all of the players in a round. You can also toggle the **Player Overlay** here, which draws some basic information (username, character name, is antag) over all players in-game.
+
+The *Objects* tab lets you view all of the grids, maps, and stations in the current round, including their name and Entity IDs. You can also teleport to an object or delete it from the tab.
+
+---
 
 ## Verbs
 
 ### Admin Verbs
 
 ![verbs.png](../../assets/images/admin/verbs.png)
 
----
-There are multiple admin verbs available. These are all fairly self-explanatory. *Message* opens the ahelp window targeted at the player, *Open Admin Notes* opens the user's notes. *Teleport Here/To* do as they say, and *Freeze* stops them from moving or interacting until they are unfrozen by an admin.
+There are multiple admin verbs available. These are all fairly self-explanatory. 
+
+- **Clone:** Clones the targeted player's current entity.
+
+- **Erase:** Deletes the targeted player's entity from the round, along with their message history. Useful for removing a character from a round entirely, and if they were spamming.
+
+- **Mark:** Places an entity into the $marked variable, a list of entities, replacing its prior value.
+
+- **Message:** Opens the ahelp window targeted at the player.
+
+- **Open Admin Notes:** Opens the user's notes. 
+
+- **Open Player Panel:** Opens a panel with the targeted player's information, such as their username, and if they are whitelisted. The panel also includes their total playtime, notes, bans, role bans, and shared connections. The admin verbs are included in the Player Panel.
 
+- **Teleport Here/To:** Self-explanatory.
+
+- *Freeze* stops them from moving or interacting until they are unfrozen by an admin.
+
+---
 ### Debug Verbs
 
 ![debug.png](../../assets/images/admin/debug.png)
 
----
 
 The debug verbs are primarily for debugging, but are also very useful for normal admins.
 
-*Control mob* gives you control of the entity, if they are sentient.
+- **Control Mob:** Gives you control of the entity, if they are sentient.
```

Comment:
```suggestion
- **Control Mob:** Gives you control of the entity.
```
The Control Mob verb doesn't care if the entity is sentient or not. It will just make it sentient if it isn't. So yes, you can roleplay as a wall!